Tests/HaCWebsiteLibTests/DatabaseUtilsTests.swift

```diff
@@ -0,0 +1,42 @@
+import XCTest
+@testable import HaCWebsiteLib
+
+class DatabaseUtilsTests: HaCWebsiteLibTestCase {
+  func testDatabaseComponentsExtraction() {
+    XCTAssertEqual(
+      URL(string: "postgres://richard:test@hac-db:5432/hac")!.databaseURLComponents,
+      DatabaseURLComponents(
+        host: "hac-db",
+        user: "richard",
+        password: "test",
+        database: "hac"
+      )
+    )
+
+    XCTAssertEqual(
```

Comment:
Is it not best to be using XCTAssertNil here? https://developer.apple.com/documentation/xctest/nil_and_non_nil_assertions

Reply:
@jaredkhan looks like this still needs to become `AssertNil` - I think there were two cases and only one has been fixed. Otherwise, I'm keen to get this in soon :)

Reply:
@CharlieCrisp Okay good to go now